lighthouse-core/gather/connections/raw.js

```diff
@@ -10,22 +10,11 @@ const Connection = require('./connection.js');
 /* eslint-disable no-unused-vars */
 
 /**
- * @interface
```

Comment:
turns out this was never working as an interface for tsc (@interface has closure support only). Making a typedef so can be imported elsewhere